doc/installation.rst.in

```diff
@@ -222,6 +222,16 @@ the ``dynesty``, ``matplotlib``,  ``scipy``, ``pypmc``, ``pyyaml``, and ``wilson
   pip install dynesty matplotlib scipy pypmc pyyaml wilson
 
 
+If you intend to build the EOS documentation, you will also need the following software:
+
+::
+
+  # for the 'Documentation Software'
+  sudo apt-get install python3-sphinx python3-sphinx-rtd-theme python3-sphinx-argparse python3-sphinxcontrib-contentui python3-nbsphinx pandoc
+  # or using pip
+  pip install sphinx sphinx_rtd_theme sphinx-argparse sphinxcontrib-contentui nbsphinx pandoc
```

Comment:
Pretty sure you need to both install `pandoc` the library with apt-get and also `pandoc` the python wrapper with pip, you can't just do pip

Reply:
Resolved. Added a cautionary note:
```
Be advised that the ``pandoc`` python package requires a system installation of ``pandoc``.
```